### DISCUSSION TITLE
what is the unused match arg in multiple functions?

### DISCUSSION THREAD

**Discussion by KRRT7** [r1chardj0n3s/parse#191] [Q&A] [answered]
Hello, trying add type hinting to this code base, however i see a match argument but i don't see it used within the functions, so i'm wondering what is it? knowing what it is would help me type it properly, currently i'm just ignoring it
here are some places where i'm seeing it
https://github.com/r1chardj0n3s/parse/blob/master/parse.py#L70
https://github.com/r1chardj0n3s/parse/blob/master/parse.py#L109
https://github.com/r1chardj0n3s/parse/blob/master/parse.py#L113
and so on.

**Comment by wimglenn** [comment, marked as answer]
It is an [`re.Match`](https://docs.python.org/3/library/re.html#re.Match) object.
